terraform/environments/cooker/platform_data.tf

```diff
@@ -171,3 +171,16 @@ data "aws_iam_session_context" "whoami" {
 data "http" "environments_file" {
   url = "https://raw.githubusercontent.com/ministryofjustice/modernisation-platform/main/environments/${local.application_name}.json"
 }
+
```

Comment:
One minor point - these should be put in the data.tf, the platform_ files are managed at a platform level, so are liable to be overwritten on bulk so shouldn't be edited unless necessary.

Reply:
OK. I will move them.